src/curvefit/models/core_model.py

```diff
@@ -56,14 +57,14 @@ def objective(self, x, data):
             param_gprior=self.data_inputs.param_gprior_info,
         )
 
-    def get_params(self, x):
+    def get_params(self, x, expand=True):
         return effects2params(
             x,
             self.data_inputs.group_sizes,
             self.data_inputs.covariates_matrices,
             self.param_set.link_fun,
             self.data_inputs.var_link_fun,
-            expand=False,
+            expand=expand
         )
```

Comment:
@jizezhang just wanted to point your attention to this modification. I did it because for some examples it's easier to request expanded params, for some -- condensed ones.

Reply:
oh, can you set default to be `False`? otherwise it will break the `predict` function below. thanks!